<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeanTestWdg</class>
 <widget class="QWidget" name="SeanTestWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeanTestWdg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <widget class="QWidget" name="contentWdg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <widget class="QPushButton" name="penBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lastBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>2</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>2</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerBtn">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>2</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pageLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>5</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1/50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
